blueprints/jsonapi-adapter/files/__root__/__path__/__name__.js

```diff
@@ -1,17 +1,5 @@
 <%= importStatement %>
-import config from '../config/environment';
 
 export default <%= baseClass %>.extend({
   type: '<%= entity %>',
-
-  url: config.APP.API_PATH + '/<%= resource %>',
```

Comment:
I think you were trying to say we should add a note about the default url. I agree. I'll update https://github.com/pixelhandler/ember-jsonapi-resources/pull/99